docker-compose.yml

```diff
@@ -92,6 +92,7 @@ services:
       - ./docker/mailhog/maildir:/maildir:rw,delegated
     ports:
       - "8025:8025"
+      - "1025:1025"
```

Comment:
This line should be removed.

Reply:
Could you please explain why? I've added it to made the mailhog service visible for the php service. Hence the other modified line (https://github.com/Sylius/Sylius-Standard/pull/334/commits/be9d23456f0a607f304edc9839fd161c04e22fb0#diff-4e5e90c6228fd48698d074241c2ba760R20).

Reply:
That is not how it works. :smile:

Reply:
Since the 2 containers are connected to the same network, they can see each other just fine. That's how the `nginx` container connects to the `php` container, for example.

Reply:
What you've actually done here with this line is to publish port 1025 from the `php` container to port 1025 on the host, which of course makes no sense.